### PR TITLE
feat(windows): select language at keyboard install

### LINKED ISSUE #1456: [Windows] Choosing a Language and then a multilingual keyboard always installs KB's 1st language.
This is a workflow issue.
Keyman 11.0.85
Choose Ewondo and Download Cameroon QWERTY or Cameroon AZERTY. It chooses the first language listed alphabetically.

**Steps:**
- Configuration>Download Keyboard>E>Ewondo>Cameroon Qwerty>Install>OK

Cameroon Keyboard is installed, but the final chosen language is Afade, I know I can add a new language and delete the old one now, but it should have remembered the selection of Ewondo.

### PR DESCRIPTION
**Pull request by mcdurdin**
For packages with a single keyboard, we now make it possible to select a language at install time.

Fixes #1456.  Part of omnibus #3512.

# Screenshots

## Single keyboard / single language, and multi keyboard

These are the same as before.

image

image

## Single keyboard / multiple languages

The combo is pre-filled with the bcp47 tag found in the search.

image

image

image

### CODE REVIEW

**Comment by darcywong00** [comment]
I wonder if the combo box would benefit from a separate bold label.

When I compare Galaxie Greek (2 keyboard layouts) and sil_euo_latin (1 keyboard layout w/ a language picker), I might confuse the dropdown to involve a second keyboard layout

File: windows/src/desktop/kmshell/install/Keyman.Configuration.System.TIPMaintenance.pas [hunk 1]
```diff
@@ -177,6 +178,13 @@ class function TTIPMaintenance.DoRegister(const KeyboardID,
     Result := False;
 end;
 
+class function TTIPMaintenance.GetFirstLanguage(Keyboard: IKeymanKeyboardFile): string;
+begin
+  if Keyboard.Languages.Count > 0
+    then Result := Keyboard.Languages[0].BCP47Code
+    else Result := TLanguageCodeUtils.TranslateWindowsLanguagesToBCP47(HKLToLanguageID(GetDefaultHKL));
```
Comment (darcywong00) [inline windows/src/desktop/kmshell/install/Keyman.Configuration.System.TIPMaintenance.pas:185]: Is this returning the default locale language tag? (which could be completely separate from the Keyboard.Languages[] list)

Comment (mcdurdin) [reply windows/src/desktop/kmshell/install/Keyman.Configuration.System.TIPMaintenance.pas:185]: Yes, `GetDefaultHKL` returns the user's default keyboard handle, and that's translated to a `LANGID` and from there into a BCP 47 tag. The ultimate fallback!

**Comment by mcdurdin** [comment]
Updated styling per review comments:

image

**Review by jahorton — APPROVED** [review]

**Pull request merged**